internal/graph/check.go

```diff
@@ -422,6 +428,11 @@ func exclusion(ctx context.Context, concurrencyLimit uint32, handlers ...CheckHa
 		}
 	}
 
+	_ = errs
```

Comment:
Remove after uncommenting the lines above. Uncommenting the lines above should resolve the reported issue.